contrib/cirrus/CIModes.md

```diff
@@ -109,8 +101,6 @@ of this document, it's not possible to override the behavior of `$CIRRUS_PR`.
 + artifacts
 
 ### Intended Branch tasks (and Cirrus-cron jobs, except "multiarch"):
-+ ext_svc_check
-+ *build*
```

Comment:
@cevich just doublechecking that you intended to remove the build task from the branch task list.

Reply:
Oh woops, no I didn't.  Thanks for catching that.